Algorithm.CSharp/SamcoBasicTemplateOptionsAlgorithm.cs

```diff
@@ -0,0 +1,162 @@
+/*
+ * QUANTCONNECT.COM - Democratizing Finance, Empowering Individuals.
+ * Lean Algorithmic Trading Engine v2.0. Copyright 2014 QuantConnect Corporation.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using QuantConnect.Data;
+using QuantConnect.Data.Market;
+using QuantConnect.Orders;
+using QuantConnect.Interfaces;
+using QuantConnect.Brokerages;
+
+namespace QuantConnect.Algorithm.CSharp
+{
+    /// <summary>
+    /// This example demonstrates how to add options for a given underlying equity security.
+    /// It also shows how you can prefilter contracts easily based on strikes and expirations, and how you
+    /// can inspect the option chain to pick a specific option contract to trade.
+    /// </summary>
+    /// <meta name="tag" content="using data" />
+    /// <meta name="tag" content="options" />
+    /// <meta name="tag" content="filter selection" />
+    public class SamcoBasicTemplateOptionsAlgorithm : QCAlgorithm
+    {
+        private const string UnderlyingTicker = "NIFTY";
+        public Symbol OptionSymbol;
+
+        public override void Initialize()
+        {
+            //SetStartDate(2015, 12, 24);
+            //SetEndDate(2015, 12, 24);
+            //SetCash(100000);
+
+            SetTimeZone(TimeZones.Kolkata);
+            SetBenchmark(t => 0);
```

Comment:
`SetBenchmark(t => 0);` is repeated twice - Believe the `Samco` brokerage model should override the default benchmark 👍 Same for the `ZerodhaBrokerageModel`